### DISCUSSION TITLE
Link to model from GET endpoints

### DISCUSSION THREAD

**Discussion by yonderblue** [scalar/scalar#1831] [Ideas]
Would be great to link to models. For example [here](https://docs.scalar.com/project-default/version-default/reference-nOvFu4SFm1hCRPclrBhuv#tag/planets/get/planets/{planetId}) it shows an example output and optionally the schema if you hit the Show Schema checkbox, but not a link over to the Model section of Planet.

**Comment by marclave** [comment]
we are on the same brain wavelength! @cameronrohani and i were chatting about this at the beginning of the week! lets make this an issue and improve it ✨

**Reply by hanspagel** [reply]
Relevant issue: https://github.com/scalar/scalar/issues/1847
